shimmingtoolbox/cli/b0shim.py

```diff
@@ -486,10 +497,20 @@ def _save_to_text_file_static(coil, coefs, list_slices, path_output, o_format, o
 @click.option('--optimizer-method', 'method', type=click.Choice(['least_squares', 'pseudo_inverse']), required=False,
               default='least_squares', show_default=True,
               help="Method used by the optimizer. LS will respect the constraints, PS will not respect the constraints")
-@click.option('--optimizer-criteria', 'opt_criteria', type=click.Choice(['mse', 'mae']), required=False,
+#########code added by Yixin############################################################################################
+@click.option('--weighting-signal-loss', 'w_signal_loss', type=click.FLOAT, required=False, default=0.0, show_default=True,
+              help="weighting for signal loss recovery. Since there is generally a compromise between B0 inhomogeneity"
```

Comment:
Add that it is only for the 'grad' optimizer

Reply:
Thanks! addresed this in https://github.com/shimming-toolbox/shimming-toolbox/commit/9c2ba526bef3781adfbd5b34d95d37b3e7f5c7da